src/components/LoadingIndicator/LoadingIndicator.tsx

```diff
@@ -0,0 +1,81 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+import styles from './LoadingIndicator.module.css';
+
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+
+  /**
+   * Layout size of the loader. This affects the overall size and associated
+   * stroke width.
+   */
+  size?: 'sm' | 'md' | 'lg';
+
+  /**
+   * Aria label of the oval. Default is oval-loading. Will be overridden if ariaLabel is passed in props
```

Comment:
Can we use a different default aria label? "oval loading" sounds pretty weird. Just "loading" would be better.

Reply:
Yah, certainly. This default comes from the library itself, but doesn't make much sense